tokio-console/src/view/mod.rs

```diff
@@ -168,32 +177,56 @@ impl View {
         update_kind
     }
 
+    fn handl_help_popup(&mut self, event: crossterm::event::Event) {
+        if input::is_help_toggle(&event) {
+            // TODO: Pause state if we are about to show the help toggle
+            self.show_help_toggle = !self.show_help_toggle;
+        }
+        if self.show_help_toggle && !input::is_help_toggle(&event) {
```

Comment:
nit, take it or leave it: if we used `if ... else if`, we could avoid the second call to `input::is_help_toggle`:
```suggestion
        } else if self.show_help_toggle {
```